added in questions (#8)
UBC-MDS/R2Py2: README.md

A Survey of:

_Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._

| Team Members | Github Handle |
|--------------|---------------|
| Brenden Everitt | everittB |
| Miliban Keyim | mkeyim |
| Maninder Kohli | ksm45 |
| Phuntsok Tseten | phuntsoktseten |

#### Chosen Questions:

> 

Author: need to ask 
'which do you prefer for data analysis'

> 

Author: ```suggestion
* Which language do you prefer for data analysis:
- Python or R. This is the response question.
```

A Survey of:

> _Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._

Reviewer: _Does the first programming language learned influence whether someone prefers Python or R when it comes to data analysis?_

> _Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._

Reviewer: ```suggestion
_Does the first programming language learned influence whether someone prefers Python or R when it comes to data analysis?_
```

* What was your undergraduate degree in? 
  - Educational background will influence what programming language one prefers to learn first.

* Are your analysis projects typically small(10K rows), medium(10-1M Rows), big (1m+ rows) ?
  - The size of a data analysis project might influence the language a person to use because of difference in speed and ease of setup/use.
  
* Which language do you think has better documentation when it comes to Data Analysis?
  - We believe having clear and concise documentation aids in the ease of applying data analysis principles and decreases the learning curve of the language. If clear examples of analysis using the languages of R or Python are not given, the user might tend to prefer to choose the one they believe has better documentation that helps them through the process.  

#### Analysis of the survey results we plan to employ.

We will use Logisitic regression as we have a binary outcome of either Python or R, with that we are able to see the effect of the confounding variables. 

#### Discuss the aspects of the UBC Office of Research Ethics document on Using Online Surveys that are relevant to your proposed survey.

> 

Reviewer 2: ```suggestion
- We will not be collecting any names in our surveys 
- Our survey will be anonymous
- Our Survey will not include any IP address that could be used to identify a person
```

> 

Reviewer 2: Aspects of the UBC Office of Research Ethics document on Using Online Surveys that is relevant to our survey are:
```
- We will not be collecting any names in our surveys 
- Our survey will be anonymous
- Our Survey will not include any IP address that could be used to identify a person

```

A Survey of:

_Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._

| Team Members | Github Handle |
|--------------|---------------|
| Brenden Everitt | everittB |
| Miliban Keyim | mkeyim |
| Maninder Kohli | ksm45 |
| Phuntsok Tseten | phuntsoktseten |

#### Chosen Questions:
* Which language do you prefer for data analysis:
- Python or R. This is the response question.
* What is the first computing language you learned?
  -  This survey question explores the influence of specific languages for the preference of Python or R.

* What was your undergraduate degree in? 
  - Educational background will influence what programming language one prefers to learn first.

* Are your analysis projects typically small(10K rows), medium(10-1M Rows), big (1m+ rows) ?

>   - The size of a data analysis project might influence the language a person to use because of difference in speed and ease of setup/use.

Reviewer: ```suggestion
  - The size of a data analysis project might influence the language a person tends to use because of difference in speed and ease of setup/use.
```

  
* Which language do you think has better documentation when it comes to Data Analysis?
  - We believe having clear and concise documentation aids in the ease of applying data analysis principles and decreases the learning curve of the language. If clear examples of analysis using the languages of R or Python are not given, the user might tend to prefer to choose the one they believe has better documentation that helps them through the process.  

#### Analysis of the survey results we plan to employ.

> We will use Logisitic regression as we have a binary outcome of either Python or R, with that we are able to see the effect of the confounding variables. 

Reviewer: ```suggestion
We plan to use logistic regression to analysis how someone's preference to use R or Python for a data analysis task is influenced by the first programming language learned they have learned, while also controlling for confounding effects. We chose a logisitic regression because we have a binary response variable, either R or Python, and will the other survery question answers as explanatory variables.
```

#### Chosen Questions:
* Which language do you prefer for data analysis:

> - Python or R. This is the response question.

Author: `
  - Python or R. This is the response question.
`
